pkg/cloudprovider/aws/instance.go

```diff
@@ -155,6 +157,9 @@ func (p *InstanceProvider) launchInstances(ctx context.Context, constraints *v1a
 	} else {
 		createFleetInput.OnDemandOptions = &ec2.OnDemandOptionsRequest{AllocationStrategy: aws.String(ec2.FleetOnDemandAllocationStrategyLowestPrice)}
 	}
+	if fleetContext != "" {
+		createFleetInput.Context = &fleetContext
```

Comment:
can you remove the `getFleetContext` func and pass the `constraints.Context` directly to the Context field in the CreateFleetInput struct? 

```
createFleetInput := &ec2.CreateFleetInput{
    Type: aws.STring(ec2.FleetTypeInstant),
    Context: constraints.Context,
....
```